smart_campus/app/static/app/css/index.css

```diff
@@ -1,3 +1,11 @@
-.segment {
-  max-width: 300px;
+.ui.borderless.center.aligned.container {
+  margin-bottom: 1em;
+  display: block;
+}
+.three.column.row {
+  margin: 0em auto;
+  margin-bottom: 2em;
+}
+#marginCard {
```

Comment:
According to your style in other CSS files, you should add a blank line between each block.

Reply:
Okay, fixed it.